portal.go

```diff
@@ -1510,7 +1510,9 @@ func (portal *Portal) convertTextMessage(intent *appservice.IntentAPI, source *U
 		}
 		expiresIn = contextInfo.GetExpiration()
 
-		extraAttrs["com.beeper.linkpreview"] = portal.convertURLPreviewToBeeper(intent, source, msg.GetExtendedTextMessage())
+		extraAttrs["com.beeper.linkpreviews"] = []*BeeperLinkPreview{
+			portal.convertURLPreviewToBeeper(intent, source, msg.GetExtendedTextMessage()),
```

Comment:
This needs to be inside an `if` now, or alternatively return an array in `convertURLPreviewToBeeper`. Otherwise we get an array with a null item inside it (null/empty array is fine though)